### PR TITLE
Captain's Quarters show up as red on player's board

### PR DESCRIPTION
**Pull request by croysdar**
image

### CODE REVIEW

File: src/main/java/assets/game.js [hunk 1]
```diff
@@ -46,6 +46,21 @@ function markHits(board, elementId, surrenderText) {
     });
 }
 
+// function markCaptains(board ){
+//     // captain quarters front end?
+//     // board.visibleSquares.captains
+//     board.visibleSquares.foreach((square) =>{
+//         // let className;
```
Comment (jtparks) [inline src/main/java/assets/game.js]: Consider taking the commented out function out of the code.

File: src/main/java/assets/game.js [hunk 2]
```diff
@@ -103,6 +119,48 @@ function cellClick() {
...
+                        document.getElementById("player").rows[square.row - 1].cells[square.column.charCodeAt(0) - 'A'.charCodeAt(0)].classList.add("captains");
+                    }
+                }
+            }
+
+            // // this is where i am working on front end captains quarters (in the cellClick function)
+            // // maybe this forEach line is not working, it seems like square comes out null
+            // game.playersBoard.ships.forEach((ship) => ship.occupiedSquares.forEach((square) => {
+            //     // for( result previousattack : this.attack)
+
```
Comment (jtparks) [inline src/main/java/assets/game.js]: Same as previous comment, consider taking out this bunch of commented code.

File: src/main/java/assets/game.js [hunk 3]
```diff
@@ -60,6 +75,34 @@ function redrawGrid() {
     }));
     markHits(game.opponentsBoard, "opponent", "winModal");
     markHits(game.playersBoard, "player", "surrModal");
+<<<<<<< HEAD
```
Comment (loganintech) [inline src/main/java/assets/game.js]: This PR has merge issues

File: src/main/java/assets/game.js [hunk 4]
```diff
@@ -60,6 +75,34 @@ function redrawGrid() {
...
+
+            // since captains is undefined, this is never running
+            // once captains is defined, then this should all work
+            if (game.playersBoard.ships[i].occupiedSquares[j].captains == true) {
+                console.log("testing captain" + j);
+                document.getElementById("player").rows[game.playersBoard.ships[i].occupiedSquares[j].row - 1].cells[game.playersBoard.ships[i].occupiedSquares[j].column.charCodeAt(0) - 'A'.charCodeAt(0)].classList.add("captains");
+            }
+        }
+    }
+=======
```
Comment (loganintech) [inline src/main/java/assets/game.js]: This PR has merge issues

File: src/main/java/assets/game.js [hunk 5]
```diff
@@ -60,6 +75,34 @@ function redrawGrid() {
...
+    // markCaptains(game.playersBoard, "player");
+>>>>>>> a840c90bd5f40076043ab9d498af72017b7ad2eb
```
Comment (loganintech) [inline src/main/java/assets/game.js]: This PR has merge issues

File: src/main/java/assets/game.js [hunk 6]
```diff
@@ -103,6 +146,51 @@ function cellClick() {
                 isSetup = false;
                 registerCellListener((e) => { });
             }
+
+<<<<<<< HEAD
```
Comment (loganintech) [inline src/main/java/assets/game.js]: This PR has merge issues

File: src/main/java/assets/game.js [hunk 7]
```diff
@@ -103,6 +146,51 @@ function cellClick() {
...
+=======
```
Comment (loganintech) [inline src/main/java/assets/game.js]: This PR has merge issues

File: src/main/java/assets/game.js [hunk 8]
```diff
@@ -103,6 +146,51 @@ function cellClick() {
...
+            //         // the document line should work, it worked in the else
+            //         document.getElementById("player").rows[square.row - 1].cells[square.column.charCodeAt(0) - 'A'.charCodeAt(0)].classList.add("captains");
+            //     }
+            //     else {
+            //         // console.log("test");
+            //         console.log("not captains quarters");
+            //         // document.getElementById("player").rows[square.row - 1].cells[square.column.charCodeAt(0) - 'A'.charCodeAt(0)].classList.add("notcaptains");
+            //     }
+            // }));
+>>>>>>> a840c90bd5f40076043ab9d498af72017b7ad2eb
```
Comment (loganintech) [inline src/main/java/assets/game.js]: This PR has merge issues

**Pull request merged**